### PR TITLE
Title from settings.py

### PR DESCRIPTION
**Pull request by sbonaime**
Instead of NEMO, we can choose the Site Title from settings.py

### CODE REVIEW

File: NEMO/models.py [hunk 1]
```diff
@@ -1674,4 +1674,4 @@ def record_active_state(request, obj, form, field_name, is_initial_creation):
 		activity_entry.authorizer = request.user
 		activity_entry.action = getattr(obj, field_name)
 		activity_entry.content_object = obj
-		activity_entry.save()
+		activity_entry.save()
```
Comment (sbonaime) [inline NEMO/models.py:1677]: I don't modify this file ! Maybe a typo or bug between Visual code and Github...

**Comment by mrampant-nist** [comment]
Hi, would you mind submitting it again with just the relevant commits (instead of those 18 commits)?
I don't want to pollute the master branch too much. thanks

**Comment by sbonaime** [comment]
> Hi, would you mind submitting it again with just the relevant commits (instead of those 18 commits)?
> I don't want to pollute the master branch too much. thanks

I think I manage to do it !

**Comment by sbonaime** [comment]
If this PR is accepted, the documentation should be a updated for this new feature

**Comment by mrampant-nist** [comment]
thanks. much better! 
I'll take a closer look later, but it might be worth looking in other places where the word NEMO is used. Like in customization, or other messages.
The question we need to ask ourselves is how far we want to take this. Is this only meant to customize the header title, or any GUI related text? 

Few examples:
- in the Jumbotron, is says 'NEMO is Offline'
- 'You don't have a NEMO account' when trying to login
- anywhere it says 'The NEMO administrator has been notified'

I have to think about it a bit

**Comment by dsbarth** [comment]
I used a customization and a context processor to do a similar thing, except to allow renaming the facility to something other than NanoFab. As Mathieu said, I did have to go through and find many instances of 'NanoFab' and replace each one. I guess that approach would also work here. Is there any advantage or disadvantage to doing this in settings instead?

Here is the relevant commit in my fork: https://github.com/dsbarth/NEMO/commit/9f65f4dbef9fc6feb37084baa332e323446ee0ed

**Comment by mrampant-nist** [comment]
That's a very good point. To me context processor is for content that needs to be added to the context. Template tags are more for formatting or processing that content.
About customization vs settings, I also prefer customization here. To me UI preferences should go in customization, and settings should be reserved more for architectural/plugin configuration.

So with regards to the functionality (customizing the title), I would agree that a context processor with customization makes more sense.

**Comment by mrampant-nist** [comment]
closing this pull request. I made the change throughout the whole application.
'NEMO.context_processors.device' needs to be replaced by 'NEMO.context_processors.base_context' in settings.py

**Pull request closed without merging**